api-docs.md

```diff
@@ -153,6 +153,7 @@ or running third-party containers where you don't have control over the Dockerfi
           Whether to enable automatic Pod Disruption Budget creation for this application.<br/>
           <br/>
             <i>Default</i>: true<br/>
+            <b>Note:</b> If the number of replicas is set to 0, a PodDisruptionBudget will not be created, even if <code>enablePDB</code> is set to true.
```

Comment:
As mentioned in-person, this will be overwritten by the api docs workflow. Comments should be put on the types itself.